File: library/src/main/java/com/pengrad/telegrambot/model/request/ReplyKeyboardHide.java
package com.pengrad.telegrambot.model.request;

import java.io.Serializable;

/**
 * stas
 * 8/4/15.
 */
@Deprecated
public class ReplyKeyboardHide extends Keyboard implements Serializable {
    private final static long serialVersionUID = 0L;

    private final boolean hide_keyboard = true;
    private final boolean selective;

    public ReplyKeyboardHide() {
        this(false);
    }

    public ReplyKeyboardHide(boolean selective) {
        this.selective = selective;
    }
}

File: core/src/main/kotlin/com/pengrad/telegrambot/model/request/ReplyKeyboardHide.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * stas
 * 8/4/15.
 */
@Deprecated("")
class ReplyKeyboardHide @JvmOverloads constructor(private val selective: Boolean = false) : Keyboard(), Serializable {

    private val hide_keyboard = true

    companion object {
        private const val serialVersionUID = 0L
    }
}
